core/src/main/resources/error/error-classes.json

```diff
@@ -592,6 +592,29 @@
       "Detected an incompatible DataSourceRegister. Please remove the incompatible library from classpath or upgrade it. Error: <message>"
     ]
   },
+  "INCOMPLETE_TYPE_DEFINITION" : {
+    "message" : [
+      "Incomplete complex type:"
+    ],
+    "subClass" : {
+      "ARRAY" : {
+        "message" : [
+          "The definition of `ARRAY` type is incomplete. You must provide an element type. For example: `ARRAY<elementType>`."
+        ]
+      },
+      "MAP" : {
+        "message" : [
+          "The definition of `MAP` type is incomplete. You must provide a key type and a value type. For example: `MAP<keyValueTypes>`."
```

Comment:
```suggestion
          "The definition of `MAP` type is incomplete. You must provide a key type and a value type. For example: `MAP<keyType, valueType>`."
```

Reply:
Sure, I can update this, but do we prefer a real example with real types (e.g. `MAP<STRING, INT>` ) or a templated example (e.g. `MAP<keyType, valueType>`) here?

Reply:
Done making it templated example 
``` 
For example: `MAP<keyType, valueType>`.
```

Reply:
OK let's use an actual data type. I didn't realize there is a parameter replacing here. 
For the example, I think we can hard code it in the error message. Why do we have to replace the error message parameter?

Reply:
Yes for this example `MAP<keyType, valueType>` we can do hardcode without using a parameter, as it won't be replaced (Only `<([a-zA-Z0-9_-]+)>` gets replaced as a parameter)

Done using actual data type.